tests/test_installer.py

```diff
@@ -16,7 +16,14 @@
 @pytest.fixture(scope="module")
 def installer(build_root: Path) -> list:
     installer = build_root / "install.sh"
-    return [Path(r"C:\Program Files\Git\bin\bash.EXE"), installer] if IS_WINDOWS else [installer]
+    # Windows has no shebang support, nor a native bash install.
+    if IS_WINDOWS:
+        # If the Windows environment has git installed, git vendors a bash we can reliably use.
+        git_bash = Path(r"C:\Program Files\Git\bin\bash.EXE")
```

Comment:
You took my meaning wrong here too. `C:\` is not god given, a user can install on `D:\` or wherever. I'm fine with what you had prior. Like I said, I'll follow-up.

Reply:
aha, and I suppose its not as simple as just plugging in e.g. `%ProgramFiles%\Git\bin\bash.EXE`?

windows is a foreign land to me, but my base assumption is/was that most people don't tend to tamper with the default - and thus good enough to hardcode.

Reply:
> windows is a foreign land to me, but my base assumption is/was that most people don't tend to tamper with the default 

I agree, buts devs are the audience here and they are exactly the people that screw with things.